CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 Changelog
 ===
 
+# 1.0
+
+Release date: Jul 18, 2018
+
+Enhancement:
+
+* [JENKINS-52044](https://issues.jenkins-ci.org/browse/JENKINS-52044) - Remove JNLP agent port from the Remoting Kafka Plugin demo.
```

Comment:
For the future, it may make sense to put major enhancements like JENKINS-51471 on the top